library/include/modules/Textures.h

```diff
@@ -1,7 +1,14 @@
 #pragma once
 
-#include "Export.h"
+#include <string>
+#include <vector>
+
 #include "ColorText.h"
+#include "Export.h"
+
+#include <SDL_surface.h>
```

Comment:
I'm still uncomfortable including a SDL header at this scope. Since we just interact with pointers to SDL_Surface objects, you can replace the `#include <SDL_surface.h>` with a forward declaration: `struct SDL_Surface;`